### PR TITLE
Update Chinese docs & some improvement

### PR DESCRIPTION
**Pull request by Littlegolden**
1. i18n for Outdated Tip
2. Sync English docs: dfa3d7b, 14c434e
3. Add default directory structure (docs/install.md)

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -109,6 +109,27 @@ You should never set any folder or file to permission level `777`, as this permi
...
+│
+├── vendor _(**flarum core, extensions**)_
+├── .nginx.conf _(**Nginx pseudo-static file**)_
+├── composer.json
+├── extend.php
+├── flarum
+├── LICENSE
+├── README.md
+└── site.php
+:::
```
Comment (Littlegolden) [inline docs/install.md:131]: Add default directory structure

File: docs/.vuepress/public/js/i18n.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            processing_tip = 'The translation of extend documents is still a work in progress, please refer to the <a>English document</a> for the latest information. If you have problems with the translation, please <a>let us know</a>.';
+            is_processing = false;
+            break;
+        case 'zh':
+            outdated_tip = '此页面的翻译可能更新不及时，最新信息请以<a>英文文档</a>为准。如果翻译有问题，请<a>告诉我们</a>。';
+            processing_tip = '当前文档仍在翻译中（进度 6/23），有关最新信息，请查阅<a>英文文档</a>。如果翻译有问题，请<a>告诉我们</a>。';
+            is_processing = true;
+            break;
+        case 'tr':
+            outdated_tip = 'The translation of this page may not be up to date, please refer to the <a>English document</a> for the latest information. If there is a problem with the translation, please <a>let us know</a>.';
```
Comment (askvortsov1) [inline docs/.vuepress/public/js/i18n.js:16]: Would it be possible to use translated text for these?

File: docs/install.md [hunk 3]
```diff
@@ -109,6 +109,27 @@ You should never set any folder or file to permission level `777`, as this permi
 
 By default Flarum's directory structure includes a `public` directory which contains only publicly-accessible files. This is a security best-practice, ensuring that all sensitive source code files are completely inaccessible from the web root.
 
+::: vue
```
Comment (askvortsov1) [inline docs/install.md:112]: I'm not sure this is the best place for this. Maybe under 'Installing' would be better? Also, this might be better in a separate PR.

File: docs/zh/troubleshoot.md [hunk 4]
```diff
@@ -1,7 +1,3 @@
-<template>
-  <outdated class="blue"></outdated>
-</template>
-
 # 故障排查
```
Comment (askvortsov1) [inline docs/zh/troubleshoot.md:1]: Could these be removed from other language files so they don't appear twice?

**Review by askvortsov1 — CHANGES_REQUESTED** [review]
Thank you for putting this together! Sorry for the delay in reviewing.

File: docs/zh/troubleshoot.md (hunk 4 above)
Comment (Littlegolden) [reply docs/zh/troubleshoot.md:1]: Other language files haven't used this template yet, so the notification will appear once.

File: docs/install.md (hunk 3 above)
Comment (Littlegolden) [reply docs/install.md:112]: sonds good, I'll do that

File: docs/.vuepress/public/js/i18n.js (hunk 2 above)
Comment (Littlegolden) [reply docs/.vuepress/public/js/i18n.js:16]: I don't speak Turkish, we can let @DursunCanPoyraz translate it later

File: docs/install.md (hunk 3 above)
Comment (Littlegolden) [reply docs/install.md]: this change has been reverted

File: docs/zh/troubleshoot.md (hunk 4 above)
Comment (askvortsov1) [reply docs/zh/troubleshoot.md:1]: Isn't it being used in the Italian version?

Comment (Littlegolden) [reply docs/zh/troubleshoot.md:1]: No, it isn't. I searched this template code in Italian doc, and found nothing.

**Review by askvortsov1 — APPROVED** [review]

**Review by davwheat — APPROVED** [review]
I haven't tested it locally, but it all looks good to me.
